articles/files_and_versioning.md

```diff
@@ -65,6 +67,58 @@ Navigate to the **Files** tab of the project you would like to add the file to.
 
 {% endtabs %}
 
+## Moving a File
+
+All the clients, programmatic and web, offer a way to move files and folders. Please note that file views and sync manifests **cannot** be used to move files. 
+
+The command line client has a parameter `mv` which can be used to move files and folders. The Python and R clients do not have a specific `move` function, but can be used to modify the `parentId` property of the file/folder to move it. In the web client, there is an option in the `Tools` menu to move files or folders. 
```

Comment:
parameter -> sub-command

Reply:
file Jira to make a move function

(oops there already is - https://sagebionetworks.jira.com/browse/SYNPY-582)